src/JS/open-and-close-modal.js

```diff
@@ -0,0 +1,26 @@
+function onCloseModal() {
+    document.body.classList.remove('is-hidden');
```

Comment:
Nie do końca rozumiem po co ten kawałek kodu, bo do body najprawdopodobniej nie będziemy dodawać klasy "is-hidden"